### ISSUE TITLE
Stage media only

### ISSUE THREAD

**Issue by stenberg** [stenberg/content-staging#72]
https://wordpress.org/support/topic/media-staging

**Comment by cjmaio** [comment]
I have this working and will be submitting a PR shortly.

**Comment by leup** [comment]
@cjmaio This is interesting. Do you have a PR ready somewhere or a fork available that I can watch ?
I am currently looking into this, not only for the basic editor, but also for custom fields as we are working a lot with ACF to add extra content.

**Comment by cjmaio** [comment]
The way my currently implementation works is that it actually moves over the media objects, so it's not tied to any post or custom post type. Let me see if I can clean it up for a PR.

**Comment by johannaruiz** [comment]
@cjmaio Do you have a branch or fork we can check out? I think you have the solution to the only big problem with this plugin.

**Comment by cjmaio** [comment]
I'm almost done with it, but this branch that I've forked:
- Shows a Post Type column when selecting posts to stage
- Properly translates the menu items between staging and production
- Includes post type attachment, with status inherit

In our environment, this has allowed us to push images. Though, it needs to be a bit more elegant... 700 images can mean a lot of pages.

https://github.com/cjmaio/content-staging/tree/github-72

**Comment by altert** [comment]
@cjmaio works great! There is an issue though - if I upload post with gallery, I can select media in batch and stage them too, but live site assigns them different IDs, so gallery gets broken on import

**Comment by cjmaio** [comment]
Yeah... that's an issue I've had. Since I don't think it's feasible to guarantee that the media IDs will be the same across environments, my thought is to rewrite the gallery shortcode during the push process. I've done it before with another product, RAMP, but I don't really like that solution. I'm open to suggestions.
